diff --git a/node_modules/@remix-run/dev/dist/compiler/compileBrowser.js b/node_modules/@remix-run/dev/dist/compiler/compileBrowser.js
index 8b3e1c2..aae8637 100644
--- a/node_modules/@remix-run/dev/dist/compiler/compileBrowser.js
+++ b/node_modules/@remix-run/dev/dist/compiler/compileBrowser.js
@@ -56,9 +56,9 @@ const getExternals = remixConfig => {
   // on node built-ins in browser bundles.
   let dependencies$1 = Object.keys(dependencies.getAppDependencies(remixConfig));
   let fakeBuiltins = module$1.builtinModules.filter(mod => dependencies$1.includes(mod));
-  if (fakeBuiltins.length > 0) {
-    throw new Error(`It appears you're using a module that is built in to node, but you installed it as a dependency which could cause problems. Please remove ${fakeBuiltins.join(", ")} before continuing.`);
-  }
+  // if (fakeBuiltins.length > 0) {
+  //   throw new Error(`It appears you're using a module that is built in to node, but you installed it as a dependency which could cause problems. Please remove ${fakeBuiltins.join(", ")} before continuing.`);
+  // }
   return module$1.builtinModules.filter(mod => !dependencies$1.includes(mod));
 };
 const writeAssetsManifest = async (config, assetsManifest) => {
